src/CheckoutSdk/Sources/SourceResponse.cs

```diff
@@ -0,0 +1,46 @@
+using Checkout.Common;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace Checkout.Sources
+{
+    /// <summary>
+    /// Defines a source response.
+    /// </summary>
+    public class SourceResponse
```

Comment:
This could inherit from `Resource` class in a similar manner as other response objects do.